docs/local.md

```diff
@@ -61,32 +70,51 @@ while [test-gslb2](https://github.com/k8gb-io/k8gb/tree/master/k3d/test-gslb2.ya
 Cluster [edgedns](https://github.com/k8gb-io/k8gb/tree/master/k3d/edge-dns.yaml) runs BIND and acts as EdgeDNS holding Delegated Zone for out test setup and answers
 on port `:1053`.
 
-```shell script
+```sh
 dig @localhost -p 1053 roundrobin.cloud.example.com +short
 ```
-Should return ***two A records*** from both clusters:
+Should return ***two A records*** from both clusters (IP addresses and order may differ):
 ```
-10.43.178.134
-10.43.75.137
+172.20.0.2
+172.20.0.5
+172.20.0.4
+172.20.0.6
+```
+
+You can verify that correct IP addresses of all the nodes in both clusters were populated:
+```sh
+for c in k3d-test-gslb{1,2}; do kubectl get no -ocustom-columns="NAME:.metadata.name,IP:status.addresses[0].address" --context $c; done
```

Comment:
`-ocustom-columns=...`

Reply:
it's not a typo, it's a valid syntax.. `-oyaml`, `-ojson`, all work :D I wanted to make it shorter because the line is pretty long (still a hair shorter than [this](https://github.com/k8gb-io/k8gb/blob/v0.8.6/docs/local.md?plain=1#L55) though). Imho it's good that it's a one-liner so that user can easily copy&paste it to cmdline. But, I can expand it to `-o custom-columns=...` if you find it more readable

Reply:
@jkremser , thx for explaining. nice..!